src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -141,6 +141,15 @@ function toggle(a, recursive) {
     } else {
         return false;
     }
+    
+    var event = doc.createEvent('Event');
+    event.initEvent(
```

Comment:
this call should be on one line.
to further enhance compat, what about wrapping the new logic in a check like: `if (doc.createEvent) {...}`?

Reply:
Updated.